### PR TITLE
Update rel attributes for target _blank links

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update rel attributes for target _blank links. 

## Justify why they are needed
Updated hardcoded rel values with Peter

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #3786** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3786?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3786?utm_source=stack-comment).

File: apps/store/src/components/ConfirmationPage/ContractCard.tsx [hunk 1]
```diff
@@ -71,7 +71,12 @@ const PendingContractCard = (props: ContractCardProps) => {
             </Text>
           </div>
           {props.url ? (
-            <ButtonNextLink href={props.url} target="_blank" onClick={handleClickOpen(props.id)}>
+            <ButtonNextLink
+              href={props.url}
+              target="_blank"
+              rel="noopener noreferrer"
```
Comment (robinandeer) [inline apps/store/src/components/ConfirmationPage/ContractCard.tsx:77]: It seems like "noreferrer" implies "noopener" (https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer)

So I don't think we need it, that's why I don't think we added it in the first place

**Review by robinandeer — COMMENTED** [review]
Nice work, just linked some docs in a comment that suggest "noopener" is not needed anymore

File: apps/store/src/components/ConfirmationPage/ContractCard.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/ConfirmationPage/ContractCard.tsx:77]: Right, let's remove it 👍

File: apps/store/src/components/ContactUs/ContactUs.tsx [hunk 2]
```diff
@@ -87,6 +87,7 @@ export const ContactUs = () => {
                   data-dd-action-name="Contact us | IOS App"
                   href={getAppStoreLink('apple', routingLocale).toString()}
                   target="_blank"
+                  rel="noopener noreferrer"
```
Comment (robinandeer) [inline apps/store/src/components/ContactUs/ContactUs.tsx:90]:
```suggestion
                  rel="noreferrer"
```

File: apps/store/src/components/ContactUs/ContactUs.tsx [hunk 3]
```diff
@@ -98,6 +99,7 @@ export const ContactUs = () => {
                   data-dd-action-name="Contact us | Android App"
                   href={getAppStoreLink('google', routingLocale).toString()}
                   target="_blank"
+                  rel="noopener noreferrer"
```
Comment (robinandeer) [inline apps/store/src/components/ContactUs/ContactUs.tsx:102]:
```suggestion
                  rel="noreferrer"
```

**Review by robinandeer — APPROVED** [review]
looks good, just added a couple of suggestions!

**Pull request merged**